docs/reference/snaps.rst

```diff
@@ -0,0 +1,76 @@
+.. _snap_reference:
+
+===============
+Snap Versions
+===============
+
+There are several checkbox snaps and channels to choose from. You can
+get a full list via ``snap info checkbox``. You can refer to the following
+to decide what you should install.
+
+.. _ref_which_snap:
+
+Picking your version
+====================
+
+As you may recall from the installation guide, when installing checkbox
+you need to pick a frontend and a backend. There are several parameters that
+may influence your choice of the best snap for your situation but in general
+there are three distinguishing factors in checkbox snaps: base, confinement and
+stability.
+
+Base
+----
+
+The base of a snap is the underlying version of the operating system
+that it uses to run. For example ``checkbox 22.04`` and ``checkbox22``
+are built on top of ``Ubuntu22.04``. This is not an 100% accurate
+explanation of what a base is but you can use this rule of thumb in your decision.
+If you want a more complete explanation of what bases are and how they are built,
+refer to this
+`blob post from the snapcraft team <https://snapcraft.io/blog/all-about-that-base>`_.
+
+In general we advise to use the version of checkbox that most closely matches the
+system that is going to run it. If you are not on an LTS release of Ubuntu or
+you are using a completely different operating system, try to match it to
+the closest release we have. For example, if you are on ``Ubuntu23.04``,
+``checkbox22`` is probably the one you will have to choose.
+
+Confinement
+-----------
+
+As you may know, or may learn more from
+`the snapcraft documentation <https://snapcraft.io/docs/snap-confinement>`_, a
+snap can either use ``strict`` or ``classic`` confinement. Checkbox has a snap
+for both models. The strict snaps are called ``ucXX``, the classic ones have a
+standard LTS name.
+
+In general we advise to use ``strict`` snap whenever possible.
+You are going to need the ``classic`` one whenever the tests you are running
```

Comment:
Should we add more use case here to point strict snap (ucXX) to Ubuntu Core and classic snap to Ubuntu Desktop as @pieqq mentioned [here](https://github.com/canonical/checkbox/pull/626/files#r1268847576)? This might help our user to choose a confinement better fit their environment.